wallet/src/de/schildbach/wallet/data/NotificationItemUserAlert.kt

```diff
@@ -0,0 +1,17 @@
+package de.schildbach.wallet.data
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+
+data class NotificationItemUserAlert(@StringRes val stringResId: Int,
+                                     @DrawableRes val iconResId: Int) : NotificationItem() {
+
+    override fun getId(): String {
+        return stringResId.toString()
+    }
+
+    override fun getDate(): Long {
+        return System.currentTimeMillis()
```

Comment:
having this use the current time will always have this notification as New, because its date will always be after the last observed notification time.

Reply:
The date should be set at the time that the object is created.